lib/rouge/lexers/cisco_ios.rb

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+# Based on/regexes mostly from Brandon Bennett's pygments-routerlexers:
+# https://github.com/nemith/pygments-routerlexers
+
+module Rouge
+  module Lexers
+    class CiscoIos < RegexLexer
+      title 'Cisco IOS'
+      desc 'Cisco IOS configuration lexer'
+      tag 'cisco_ios'
+      filenames '*.cfg'
+      mimetypes 'text/x-cisco-conf'
+
+      state :root do
+        rule %r/^!.*/, Comment::Single
+
+        rule %r/^(version\s+)(.*)$/ do
+          groups Keyword, Num::Float
+        end
+
+        rule %r/(desc*r*i*p*t*i*o*n*)(.*?)$/ do
+          groups Keyword, Comment::Single
+        end
+
+        rule %r/^(inte*r*f*a*c*e*|controller|router \S+|voice translation-\S+|voice-port|line)(.*)$/ do
```

Comment:
**suggestion:** Similar to the above suggestion, can we match the exact text, ie. `interface`? 🤔

Reply:
Reply above is relevant.